src/Mailgun/Api/Pagination.php

```diff
@@ -22,12 +22,52 @@
     abstract protected function safeDeserialize(ResponseInterface $response, $className);
 
     /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationNext($response)
+    {
+        return $this->getPaginationUrl($response->getNextUrl(), get_class($response));
+    }
+
+    /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationPrevious($response)
```

Comment:
Just an idea: What do you think of renaming the functions to `paginationPrevious`?